packages/apps/reputation-oracle/server/src/logger/index.ts

```diff
@@ -0,0 +1,25 @@
+import { createLogger } from './pino-logger';
+import NestLogger from './nest-logger';
+import { LogLevel } from './types';
+
+const environment = process.env.NODE_ENV || 'local';
```

Comment:
We're not actively using NODE_ENV but I think it might be good to have a enum with all possible values instead of hardcoded strings. This is not necessarily for this PR, just to keep in mind.